src/testUtil/TestCaseRecorder.ts

```diff
@@ -186,7 +185,7 @@ export class TestCaseRecorder {
             undefined,
             ["clipboard"],
             this.active ? this.extraSnapshotFields : undefined,
-            getActiveTextEditor()!,
+            ide().activeTextEditor!,
```

Comment:
Tempted to let snapshot code get this off ide obj

Reply:
Right now I just did a find and replace